Wizardry/Core/SkillSys/MiscSkillEffects/MenuSkills/MiscSkillEffects.event

```diff
@@ -0,0 +1,51 @@
+#include "PostActionSkills/AlertStance.lyn.event"
```

Comment:
I think this file should not be modified like this?

Reply:
Well the lyn files are needed here. I did state originally that I wasn't sure if seperate C files were needed for each rally skill or not. I wasn't sure of the best practice in this scenario.

Reply:
![image](https://github.com/user-attachments/assets/6ec346b8-1f5a-4510-b2e5-5bcb184ab617)
so what is this

Reply:
Fine I have identified this

Reply:
Sorry Mokha, I'm not sure what you're asking me, have I done something wrong?

My understanding has been the lyn files need to be included here to generate them for the rally skills.